fhir-path/src/test/java/com/ibm/fhir/path/test/MemberOfFunctionTest.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * (C) Copyright IBM Corp. 2019
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.path.test;
+
+import static com.ibm.fhir.path.evaluator.FHIRPathEvaluator.*;
+import static com.ibm.fhir.model.type.String.string;
+
+import java.util.Collection;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import com.ibm.fhir.model.type.Code;
+import com.ibm.fhir.model.type.CodeableConcept;
+import com.ibm.fhir.model.type.Coding;
+import com.ibm.fhir.model.type.Uri;
+import com.ibm.fhir.path.FHIRPathNode;
+import com.ibm.fhir.path.evaluator.FHIRPathEvaluator;
+
+public class MemberOfFunctionTest {            
```

Comment:
any javadoc worth adding here?  maybe high-level overview of the function (copy from operationDefinition?) with highlights for any "decisions" you made (any ambiguity in spec?).  for example, it might be a good spot to summarize the high-level algorithm for matching concepts in versioned systems...like whether we behave differently for systems where versionNeeded is true vs false (e.g. see https://chat.fhir.org/#narrow/stream/179202-terminology/topic/CodeSystem.2EversionNeeded)

Reply:
My plan is to create a README.md in the repo and/or on the GH pages site that explains the capability in more detail with some examples.

Reply:
FYI I intended for this comment to be on MemberOfFunction, not MemberOfFunctionTest

Reply:
Added some additional javadoc to `MemberOfFunction` class.